ff/src/fields/mod.rs

```diff
@@ -188,6 +188,32 @@ pub trait Field:
         }
         res
     }
+
+    /// Exponentiates this element by a number represented with `u64` limbs,
+    /// least significant limb first.
+    #[inline]
+    fn pow_with_table<S: AsRef<[u64]>>(&self, exp: S, pows_2: &[Self]) -> Self {
+        let exp = exp.as_ref();
+        let len = exp.len();
+        let mut max = len * 64;
+        for i in 0..len {
+            let lz = exp[len - 1 - i].leading_zeros() as usize;
+            max -= lz;
+            if lz == 64 {
+                break;
+            }
+        }
```

Comment:
Can you add a comment describing what this is doing?

Reply:
I changed it to use the BitIteratorLE